Prox/Prox/Utilities/EventNotificationsManager.swift

```diff
@@ -151,12 +151,15 @@ class EventNotificationsManager {
                     center.add(request) { error in
                         if let theError = error {
                             NSLog(theError.localizedDescription)
+                            Analytics.logEvent(event: AnalyticsEvent.ERROR_STATE, params: [AnalyticsEvent.PARAM_ACTION: AnalyticsEvent.EVENT_NOTIFICATION])
```

Comment:
It'd be great to capture the exact error that happens here (like how we capture the permissions error). Not sure it's worth doing now, but perhaps for future reference.